### ISSUE TITLE
True Color

### ISSUE THREAD

**Issue by Gabirel** [Gabirel/Hack-SpaceVim#7]
What is true color?

Why SpaceVim choose to use true colors by default?

Come from: https://github.com/Gabirel/Hack-SpaceVim/issues/4#issuecomment-293269621

**Comment by Gabirel** [comment]
![ture-color-failure](https://cloud.githubusercontent.com/assets/12933851/25032871/9013578c-210a-11e7-9b4e-f39dee4e1a01.jpg)

**Comment by Gabirel** [comment]
For windows user:
`xshell` doesn't support true colors.

Someone has already asked this feature. `xshell` maybe support true color in the future version,

If you really want to use SpaceVim in `xshell`, **please use SpaceVim in `tmux` in your `xshell`**.

**Note: tmux version >= 2.3**

**Comment by Gabirel** [comment]
## For MacOS users:

Please use `item2` instead of `iterm`

**Comment by ourml** [comment]
![image](https://user-images.githubusercontent.com/19163619/27833549-327246de-6088-11e7-8b42-e4b1d46bb75f.png)

**Comment by Gabirel** [comment]
@ourml Please give more information about this error, such as how to execute it or something

**Comment by ourml** [comment]
![capture](https://user-images.githubusercontent.com/19163619/27833724-d2065fd2-6088-11e7-9a31-ae77f905e972.PNG)



when I click Enter, it shows lots of code like the that picture

**Comment by Gabirel** [comment]
@ourml You could try `bash test_24bit.sh` instead of `./test_24bit.sh` if your OS is Ubuntu or Debian

**Comment by ourml** [comment]
it's the same problem

**Comment by ourml** [comment]
my OS is Unbuntu Server, and it's running on virtual environment(The software 'VMware Workstation'), is that related to it?

**Comment by Gabirel** [comment]
No, I don't think so. I guess your shell is `zsh`. However, `zsh` works for me though.

![image](https://user-images.githubusercontent.com/12933851/27834032-e36539fe-6107-11e7-8a06-7a0f99854fdc.png)


One thing needs to pay attention is that Ubuntu's shell uses `dash` as default. This could result in this kind of error. It's weird if you use `bash` to execute it.

@wsdjeg Have you ever heard of this weird thing?

**Comment by ourml** [comment]
Yes, you are right, my shell is `zsh`, I try it agin

**Comment by Gabirel** [comment]
Or, can you just go into `bash` and then execute it again?

**Comment by ourml** [comment]
I tried, but it's the same, even use the root to execute

**Comment by Gabirel** [comment]
This problem has been fixed.

I will update the latest version of test script to make sure that everyone could use it without any errors.

**Comment by ourml** [comment]
Thanks
